app/renderer/components/config_menu.js

```diff
@@ -58,16 +58,18 @@ module.exports = {
                     extensions: ['json']
                 }]
             }, (fileNames) =>{
-              if (!fileNames) return;
-              if (!fileNames[0]) return ;
-              const fileName = fileNames[0];
-              const confirmationAlert = new DeleteConfirmationAlert("You will not be able to recover this task after deletion!");
-              confirmationAlert.toggle().then(() => {
-                TaskConfig.clearTasks();
-                fs.readFile(fileName, 'utf-8', (err, data) =>{
-                  TaskConfig.loadTasksFrom(data);
-                });
-            }, () => {});
+              if (fileNames){
```

Comment:
`if (filenames && filenames[0])` better

Also, filename instead of fileName please